Lonchanick.HabbitLogger/Lonchanick.HabbitLogger/Program.cs

```diff
@@ -0,0 +1,80 @@
+﻿
+
+
+
+
+using Lonchanick.HabbitLogger;
+
+string aux;
+int op;
```

Comment:
I'm not sure what `aux` and `op` stand for. Maybe you can make these variables more verbose, it's okay to use very long names. Something like `menuOptionInputByUser` is very long, but it would tell the reader what the variable is for.